seacrowd/sea_datasets/limesoda/limesoda.py

```diff
@@ -0,0 +1,169 @@
+
+import datasets
+
+from seacrowd.utils import schemas
+from seacrowd.utils.configs import SEACrowdConfig
+from seacrowd.utils.constants import Licenses, Tasks
+
+_CITATION = """\
+@INPROCEEDINGS{9678187,
+  author={Payoungkhamdee, Patomporn and Porkaew, Peerachet and Sinthunyathum, Atthasith and Songphum, Phattharaphon and Kawidam, Witsarut and Loha-Udom, Wichayut and Boonkwan, Prachya and Sutantayawalee, Vipas},
+  booktitle={2021 16th International Joint Symposium on Artificial Intelligence and Natural Language Processing (iSAI-NLP)},
+  title={LimeSoda: Dataset for Fake News Detection in Healthcare Domain},
+  year={2021},
+  volume={},
+  number={},
+  pages={1-6},
+  doi={10.1109/iSAI-NLP54397.2021.9678187}}
+"""
+
+_DATASETNAME = "limesoda"
+
+_DESCRIPTION = """\
+Thai fake news dataset in the healthcare domain consisting of curate and manually annotated 7,191 documents
+(only 4,141 documents contain token labels and are used as a test set of the baseline models).
+Each document in the dataset is classified as fact, fake, or undefined.
+"""
+
+_HOMEPAGE = "https://github.com/byinth/LimeSoda"
+
+_LICENSE = Licenses.CC_BY_4_0.value
+
+_URLS = {
+    "split": {
+        "train": "https://raw.githubusercontent.com/byinth/LimeSoda/main/dataset_train_wo_tokentags_v1/train_v1.jsonl",
+        "val": "https://raw.githubusercontent.com/byinth/LimeSoda/main/dataset_train_wo_tokentags_v1/val_v1.jsonl",
+        "test": "https://raw.githubusercontent.com/byinth/LimeSoda/main/dataset_train_wo_tokentags_v1/test_v1.jsonl",
+    },
+    "raw": "https://raw.githubusercontent.com/byinth/LimeSoda/main/LimeSoda/Limesoda.jsonl",
+}
+
+_SUPPORTED_TASKS = [Tasks.HOAX_NEWS_CLASSIFICATION]
+
+_SOURCE_VERSION = "1.0.0"
+
+_SEACROWD_VERSION = "1.0.0"
+
+
+class LimesodaDataset(datasets.GeneratorBasedBuilder):
+    SOURCE_VERSION = datasets.Version(_SOURCE_VERSION)
+    SEACROWD_VERSION = datasets.Version(_SEACROWD_VERSION)
+
+    BUILDER_CONFIGS = [
+        SEACrowdConfig(
+            name="limesoda_source",
+            version=SOURCE_VERSION,
+            description="limesoda source schema",
+            schema="source",
+            subset_id="limesoda",
+        ),
+        SEACrowdConfig(
+            name="limesoda_split_source",
+            version=SOURCE_VERSION,
+            description="limesoda source schema",
+            schema="source",
+            subset_id="limesoda_split",
+        ),
+        SEACrowdConfig(
+            name="limesoda_seacrowd_text",
+            version=SEACROWD_VERSION,
+            description="limesoda SEACrowd schema",
+            schema="seacrowd_text",
+            subset_id="limesoda",
+        ),
+        SEACrowdConfig(
+            name="limesoda_split_seacrowd_text",
+            version=SEACROWD_VERSION,
+            description="limesoda: split SEACrowd schema",
+            schema="seacrowd_text",
+            subset_id="limesoda_split",
+        ),
+    ]
+
+    DEFAULT_CONFIG_NAME = "limesoda_source"
```

Comment:
let's use `_DATASETNAME` variable
```suggestion
            name=f"{_DATASETNAME}_source",
            version=SOURCE_VERSION,
            description="limesoda source schema",
            schema="source",
            subset_id=_DATASETNAME,
        ),
        SEACrowdConfig(
            name=f"{_DATASETNAME}_split_source",
            version=SOURCE_VERSION,
            description="limesoda source schema",
            schema="source",
            subset_id=f"{_DATASETNAME}_split",
        ),
        SEACrowdConfig(
            name=f"{_DATASETNAME}_seacrowd_text",
            version=SEACROWD_VERSION,
            description="limesoda SEACrowd schema",
            schema="seacrowd_text",
            subset_id=_DATASETNAME,
        ),
        SEACrowdConfig(
            name=f"{_DATASETNAME}_split_seacrowd_text",
            version=SEACROWD_VERSION,
            description="limesoda: split SEACrowd schema",
            schema="seacrowd_text",
            subset_id=f"{_DATASETNAME}_split",
        ),
    ]

    DEFAULT_CONFIG_NAME = f"{_DATASETNAME}_source"
```